sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1020,7 +1039,12 @@ def task_worker(self, stopping_event, sfp_error_event, y_cable_presence):
         while not stopping_event.is_set():
             next_state = state
             time_start = time.time()
+            # Ensure not to block for any event if sfp insert event is pending
+            if self.sfp_insert_events:
+                timeout = 1
```

Comment:
check timeout in msec or sec. its too small

Reply:
As per https://github.com/Azure/sonic-platform-common/blob/4533f829781f9d3706406f33d82a58adef74d294/sonic_platform_base/chassis_base.py#L553 definition, timeout should be in milliseconds.